### PR TITLE
Checkbox admin sessions filter

### PR DESCRIPTION
**Pull request by DhariniJeeva**
This pr adds `checkbox group` to the toolbar component on sessions admin.  onChange functionality will be addressed in the upcoming PR

### CODE REVIEW

**Review by calcaide — CHANGES_REQUESTED** [review]
Notice you are submitting `ui/admin/package-lock.json`. This file does not existed before, reason why is because we run `yarn install` within the root of the project. So `boundary-ui/yarn.lock` is the one we consider the source of truth.

I also think, you ran `npm install` rather than `yarn install` that's why it created a `package-lock.json` rather than `yarn.lock`.

**Comment by DhariniJeeva** [comment]
> yarn install



> Notice you are submitting `ui/admin/package-lock.json`. This file does not existed before, reason why is because we run `yarn install` within the root of the project. So `boundary-ui/yarn.lock` is the one we consider the source of truth.
> 
> I also think, you ran `npm install` rather than `yarn install` that's why it created a `package-lock.json` rather than `yarn.lock`.

you are right. my bad. fixing this

File: ui/admin/package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-template-lint": "^2.0.2",
     "ember-cli-terser": "^4.0.2",
+    "ember-composable-helpers": "^4.5.0",
```
Comment (calcaide) [inline ui/admin/package.json:66]: Nice! We are not seeing a change in `yarn.lock` because we already add `ember-composable-helpers: ^4.5.0` So all good!

**Review by calcaide — APPROVED** [review]
LGTM

File: ui/admin/app/controllers/scopes/scope/sessions/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+export default class ScopesScopeSessionsIndexController extends Controller {
+  // =services
+
+  @service intl;
+  @tracked selectedItems;
```
Comment (gsusmi) [inline ui/admin/app/controllers/scopes/scope/sessions/index.js:8]: `items` doesn't fit quite right. Thoughts on `selectedFilters` or even just `filters`?

File: ui/admin/app/templates/scopes/scope/sessions/index.hbs [hunk 3]
```diff
@@ -22,7 +22,23 @@
               <Rose::Dropdown
                 @text={{t 'form.status.label'}}
                 @ignoreClickInside={{true}}
-              />
+                as |dropdown|
+              >
+                <Rose::Form::Checkbox::Group
+                  @items={{this.availableSessions}}
+                  @selectedItems={{this.selectedItems}}
+                  @onChange={{this.checkboxGroupChanged}}
```
Comment (gsusmi) [inline ui/admin/app/templates/scopes/scope/sessions/index.hbs:30]: Is `checkboxGroupChanged` already defined? If this is a route action, we should call out that it's related to filters somehow. Thoughts on changing fn name to `onFilterSelection` or `filterUpdate`?

File: ui/admin/app/controllers/scopes/scope/sessions/index.js (hunk 2 above)
Comment (DhariniJeeva) [reply ui/admin/app/controllers/scopes/scope/sessions/index.js:8]: filters sound good! I'll update

File: ui/admin/app/templates/scopes/scope/sessions/index.hbs (hunk 3 above)
Comment (DhariniJeeva) [reply ui/admin/app/templates/scopes/scope/sessions/index.hbs:30]: this is a route action and can change to a better sounding name. We can go with `filterUpdate` to be in parity with other route actions

File: ui/admin/app/routes/utils/setupFilters.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export function setupFilters(controller, route, queryParam) {
+  const item = controller.controllerFor(route)[`filter-${queryParam}`];
+  return item ? JSON.parse(item) : null;
```
Comment (calcaide) [inline ui/admin/app/routes/utils/setupFilters.js:3]: Is this return working?

Assuming `item` is a `string` or `object`. I recreate this return with an `if` statement and did not work for me, maybe I am missing something.

**Comment by randallmorey** [comment]
Closing in favour of newer PRs.

**Pull request closed without merging**